### DISCUSSION TITLE
Turn off loggin on 'runs' folder

### DISCUSSION THREAD

**Discussion by sscalvo** [ultralytics/yolov5#7719] [General]
Hi there
I am launching YOLO to process thousands of images, in this way:
**model = torch.hub.load('ultralytics/yolov5', 'yolov5s')**
How can i avoir YOLO to dump content on 'runs' folder (in order to save disk space) ?
Thank you!

**Comment by glenn-jocher** [comment]
@sscalvo I don't understand your question. YOLOv5 models don't save anything unless you specifically run a results.save() command.

**Reply by sscalvo** [reply]
Thanks Glenn
Well, definetely i am not doing any results.save()
I am only read-accesing the values that YOLO finds in order to populate a database.

I dont expect you to debug my code, but just to confirm that i am not asking yolo to save anything.. The only thing my python code does related to YOLO is:

       model = torch.hub.load('ultralytics/yolov5', 'yolov5s')
        ...
        results = model(img) 
        for item in results.crop(): 
              # read item properties

and in the console i get this outputs for each image:

**Saved 1 image to runs\detect\exp227
Saved results to runs\detect\exp227**

I can live with that, but that means i have to delete the runs folder every now and then.. 
Thanks for your patience

**Reply by glenn-jocher** [reply]
@sscalvo maybe you should read the function you are calling rather than asking me
https://github.com/ultralytics/yolov5/blob/aa7a0e91b175a9b2c64c3eca44dcafe77dc23d03/models/common.py#L658-L661

**Reply by sscalvo** [reply]
there you go! Thank you very much!  👍🙏

**Comment by dipkumar** [comment]
this command line should do it:

**python detect.py --nosave --exist-ok**
